FantasmoSDK/Classes/Network/FMRestClient.swift

```diff
@@ -55,17 +31,18 @@ struct FMRestClient {
     ///   - error: Error closure
     static func post(_ endpoint: FMApiRouter.ApiEndpoint,
                      parameters: [String : String],
-                     imageData: Data,
+                     imageData: Data? = nil,
```

Comment:
This is unrelated to the task at hand.

Reply:
The meaning and reason of this change is explained in this comment - https://github.com/fantasmoio/Fantasmo-iOS-SDK/pull/29#discussion_r670308966

Small problems are noticed when working on tasks and they are quickly forgotten afterwards. Creating new tasks for them is not reasonable, ignoring them is also bad thing. As compromise this small improvements are performed immediately on noticing them.

Do you have any suggestion how it can be done better?